xnmt/linear.py

```diff
@@ -1,12 +1,26 @@
 import dynet as dy
+from xnmt.initializer import LeCunUniform as linear_init
+
 
 class Linear(object):
   def __init__(self, input_dim, output_dim, model):
-    self.W1 = model.add_parameters((output_dim, input_dim))
+    self.W1 = model.add_parameters((output_dim, input_dim), init=linear_init(input_dim))
```

Comment:
This is again changing the default behavior of the entire xnmt toolkit without making it an option. Make sure these things are set as options.